packages/model/src/Model.php

```diff
@@ -139,7 +139,7 @@ public static function migrate(?string $locale = null): void
 
         $data->transform(function (string $line) use ($schema): Collection {
             return $schema->combine(
-                array_map(fn ($value) => $value !== '' ? $value : null, str_getcsv($line))
+                array_map(fn ($value) => $value !== '' ? $value : null, str_getcsv($line, escape: "\\"))
```

Comment:
Does this still work?

```suggestion
                array_map(fn ($value) => $value !== '' ? $value : null, str_getcsv($line, escape: '\\'))
```